integration_tests/test_suites/celery-k8s-integration-test-suite/test_integration.py

```diff
@@ -582,16 +583,15 @@ def _get_step_events(event_logs):
 def test_memoization_on_celery_k8s(  # pylint: disable=redefined-outer-name
     dagster_docker_image, dagster_instance, helm_namespace
 ):
+    ephemeral_prefix = str(uuid.uuid4())
```

Comment:
does this mean our s3 usage starts to grow unboundedly? is it small enough for us to not care? Worth tryign to clean it up afterwards?

Reply:
We still do clean it up afterwards